docs/api-reference/uiSchema.md

```diff
@@ -218,5 +234,40 @@ const uiSchema = {
   "ui:title": "Your password"
 };
 ```
+
+## submitButtonOptions
+
+Sometimes it's convenient to change the behavior of the submit button for the form. This is the purpose of the `ui:submitButtonOptions` uiSchema directive:
+
+You can pass any other prop to the submit button if you want, by default, this library will set the following options / props mentioned below for all submit buttons:
+
+### `disabled` prop
+
+You can use this option to disable the submit button
+
+### `className` prop
```

Comment:
We should remove `disabled`, `className` sections because these are all now part of the `props` property. We should also add a section that says

```suggestion
### `props` prop
```

and describes what it does.